python/tvm/contrib/debugger/debug_executor.py

```diff
@@ -68,7 +73,7 @@ def create(graph_json_str, libmod, device, dump_root=None):
             "Please set '(USE_PROFILER ON)' in " "config.cmake and rebuild TVM to enable debug mode"
         )
     func_obj = fcreate(graph_json_str, libmod, *device_type_id)
-    return GraphModuleDebug(func_obj, dev, graph_json_str, dump_root)
+    return GraphModuleDebug(func_obj, dev, graph_json_str, dump_root, iters_num, repeat)
```

Comment:
Can you also add `min_repeat_ms` to the parameters. It is the 3rd parameter passed to `self.run_individual`.

Reply:
Yeah, no problem. Thanks for your suggestions.